hadoop-hdds/docs/content/feature/Topology.md

```diff
@@ -23,86 +23,190 @@ summary: Configuration for rack-awarness for improved read/write
   limitations under the License.
 -->
 
-Ozone can use topology related information (for example rack placement) to optimize read and write pipelines. To get full rack-aware cluster, Ozone requires three different configuration.
-
- 1. The topology information should be configured by Ozone.
- 2. Topology related information should be used when Ozone chooses 3 different datanodes for a specific pipeline/container. (WRITE)
- 3. When Ozone reads a Key it should prefer to read from the closest node. 
-
-<div class="alert alert-warning" role="alert">
-
-Ozone uses RAFT replication for Open containers (write), and an async replication for closed, immutable containers (cold data). As RAFT requires low-latency network, topology awareness placement is available only for closed containers. See the [page about Containers]({{< ref "concept/Containers.md">}}) about more information related to Open vs Closed containers.
-
-</div>
-
-## Topology hierarchy
-
-Topology hierarchy can be configured with using `net.topology.node.switch.mapping.impl` configuration key. This configuration should define an implementation of the `org.apache.hadoop.net.CachedDNSToSwitchMapping`. As this is a Hadoop class, the configuration is exactly the same as the Hadoop Configuration
-
-### Static list
-
-Static list can be configured with the help of ```TableMapping```:
-
-```XML
+Apache Ozone uses topology information (e.g., rack placement) to optimize data access and improve resilience. A fully rack-aware cluster needs:
+
+1.  Configured network topology.
+2.  Topology-aware DataNode selection for container replica placement (write path).
+3.  Prioritized reads from topologically closest DataNodes (read path).
+
+## Applicability to Container Types
+
+Ozone's topology-aware placement strategies vary by container replication type and state:
+
+* **RATIS Replicated Containers:** Ozone uses RAFT replication for Open containers (write), and an async replication for closed, immutable containers (cold data). As RAFT requires low-latency network, topology awareness placement is available only for closed containers. See the [page about Containers](concept/Containers.md) about more information related to Open vs Closed containers.
```

Comment:
I believe this is not true,  pipeline creation considers topology.

Reply:
hdds.scm.pipeline.choose.policy.imp default is RandomPipelineChoosePolicy, same for EC: hdds.scm.ec.pipeline.choose.policy.impl.


    /**
     * Random choose policy that randomly chooses pipeline.
     * That are we just randomly place containers without any considerations of
     * utilization.
     */
    public class RandomPipelineChoosePolicy implements PipelineChoosePolicy {

Reply:
Got it. good to know.